src/graph.rs

```diff
@@ -465,12 +520,16 @@ pub(crate) fn parse_module_from_ast(
   if module.maybe_types_dependency.is_none() {
     if let Some(headers) = maybe_headers {
       if let Some(types_header) = headers.get("x-typescript-types") {
-        let resolved_dependency = resolve(
-          types_header,
-          &module.specifier,
-          &ast::Range::default(),
```

Comment:
Removed the default implementation for something more explicit here. Also, we need to provide the specifier now, so not necessary.